src/main/java/frc/team166/robot/subsystems/Manipulator.java

```diff
@@ -119,6 +124,14 @@ public Command OpenInnerManipulator() {
         return new ActionCommand("Open Inner Manipulator", this, this::openInnerManipulator);
     }
 
+    public Command CloseInnerManipulator() {
+        return new ActionCommand("Close Inner Manipulator", this, this::closeInnerManipulator);
+    }
+
+    public Command DropCube() {
+        return new ActionCommand("Drop Cube", this, this::openInnerManipulator);
+    }
+
     public Command CubePickup() {
```

Comment:
Ben asked if this kind of thing will work: setting the motors to intake during initialization, and then not setting them in execute. Will that work? Or do you need to keep setting them in execute, not just in initialization?

Reply:
When testing this today it did work correctly. I just looked and it turns out the motor safety stuff is only used on drive objects by default: [Motor Safety](https://wpilib.screenstepslive.com/s/currentCS/m/java/l/599705-using-the-motor-safety-feature)